package/doc/sphinx/source/documentation_pages/loading_files.rst

```diff
@@ -0,0 +1,140 @@
+##################
+Loading your data
+##################
+
+   
+Basic Usage
+===========
+
+
+MDAnalysis aims to read any and all molecular simulation data,
+from whatever files you provide to it.
+This is done via the `Universe` object,
+which accepts paths to files as its arguments.
+Usually two files are required to create a `Universe` object:
+A topology file, which provides information about the names and types of atoms,
+and a trajectory file, which gives the positions of atoms over time.
+
+
+This generally corresponds to the file required to set up the simulation acting as the topology,
+while the results file of the simulation provides the trajectory.
+For example to load results from a CHARMM simulation,
+we provide a path to the PSF file to act as a topology,
+and a path to the DCD results to act as the trajectory
+::
+
+   import MDAnalysis as mda
+
+   u = mda.Universe('adk.psf', 'adk_dims.dcd')
+
+
+.. note:: If a file which also provides coordinates is used as a topology, no trajectory
+	  information is read from this file.  Ie the first frame will come from the trajectory
+	  unless the `all_coordinates` keyword is set to ``True``. 
+
+   
+Single file Universes
+---------------------
+
+Occaisionally a file may contain both topology and trajectory information,
+in such cases it is sufficient to provide only a single filename to Universe
+::
+
+   import MDAnalysis as mda
+
+   u = mda.Universe('myfile.pdb')
+
+
+Concatenating trajectories
+--------------------------
+   
+It is also possible to read multiple consecutive trajectories,
+(for example if a simulation was restarted),
+by passing a list of trajectory filenames.
+In this example, frames from `traj1.trr` and `traj2.trr` will be concatenated when iterating through the trajectory.
+::
+
+   import MDAnalysis as mda
+
+   u = mda.Universe('topol.tpr', ['traj1.trr', 'traj2.trr'])
```

Comment:
Add a SeeAlso for the ChainReader – I know that this is technical but it is useful to know where to look regarding the details of how trajectories are concatenated, especially with #1728 in the works.